src/components/Table/Table.jsx

```diff
@@ -12,7 +12,7 @@ import style from './style.scss';
 type Props = {|
   clickable?: boolean,
   empty: boolean,
-  emptyText: string,
+  emptyText: React$Node,
```

Comment:
Much better, thanks! `React$Node` also already includes a string so this just expands the type and shouldn't affect public API in any way.